### PR TITLE
1192 E2E Scenario 1 

### PR DESCRIPTION
**Pull request by wbyoko**
https://github.com/flexion/ef-cms/issues/1192#issuecomment-502137868

Scenario 1

AC - Only cases with status “General Docket - At Issue (Ready for Trial)” with a designated place of trial are eligible to be auto scheduled for trial for that city’s session

    Create trial session for [trial location]

    Create cases
        Case with status “General Docket - Not at Issue” for [trial location]
        Case with Status “General Docket - At Issue” for [trial location]
        Case with status “General Docket - At Issue” for [another trial location]

Result: Case '2' should show up as eligible for [trial location] session

### CODE REVIEW

File: web-client/e2e/trialSessionEligibleCasesScenario1.test.js [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+      COUNTRY_TYPES,
+      DOCUMENT_TYPES_MAP: Document.initialDocumentTypes,
+      INTERNAL_CATEGORY_MAP,
+      PARTY_TYPES,
+      STATUS_TYPES,
+      TRIAL_CITIES,
+    });
+  });
+
+  const trialLocation = `Baton Rouge, Louisiana, ${Date.now()}`;
```
Comment (rachaelparris) [inline web-client/e2e/trialSessionEligibleCasesScenario1.test.js:99]: Why did you add the date in here?

**Review by rachaelparris — APPROVED** [review]

File: web-client/e2e/trialSessionEligibleCasesScenario1.test.js (hunk 1 above)
Comment (wbyoko) [reply web-client/e2e/trialSessionEligibleCasesScenario1.test.js:99]: so that the trailLocation is unique.
